src/marqo/core/schema_validation.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
```

Comment:
Can you create a module validation in core and put your functions in a class `Validation`? Check the `core->document->document` as an example.